toxics/bandwidth_test.go

```diff
@@ -0,0 +1,119 @@
+package toxics_test
```

Comment:
All the tests are in a `_test` package now. I think this is prefered since we don't actually need to access internal state in these tests.